src/language-js/printer-estree.js

```diff
@@ -4024,7 +4024,9 @@ function printArgumentsList(path, options, print) {
       somePrintedArgumentsWillBreak ? breakParent : "",
       conditionalGroup(
         [
-          !somePrintedArgumentsWillBreak
+          !somePrintedArgumentsWillBreak &&
+          !node.typeArguments &&
+          !node.typeParameters
```

Comment:
You added two things here, but only one test. Does the test cover both? I can’t tell by looking at the test because I’m unsure what the difference between `typeArguments` and `typeParameters` is.

Reply:
It's a discrepancy between the AST formats. The same thing is called `typeArguments` by Flow and `typeParameters` by `typescript`. The test runs for both parsers.